sorts/merge_sort_v3.py

```diff
@@ -0,0 +1,49 @@
+# Python program for implementation of MergeSort 
+def mergeSort(arr): 
```

Comment:
No type hints or doc tests...  See CONTRIBUTING.md

Reply:
yeah  i will add that